src/components/Question.css

```diff
@@ -0,0 +1,19 @@
+.question {
+    border: 3px solid #555555;
+    border-radius: 8px;
+    background-color: #F3F3F3;
+    margin: 30px;
+    padding: 20px;
+}
+
+.option {
+    display: block;
+    border: none;
+    background-color: #FFFFFF;
+    color: #555555;
+    padding: 0px;
+}
+
+.option:hover {
+    background-color: #FFFFFF;
+}
```

Comment:
Let's do 
```
.option:hover {
    background-color: #FFFFFF;
    color: #C82B2B;
}
```

Reply:
also I forgot to do this, but if you can update the `button` styling in `App.css` to
```
button {
  font-size: 36px;
  background-color: #C82B2B;
  border: 3px solid #555555;
  color: white;
  padding: 10px 20px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  margin: 40px;
  border-radius: 8px;
  cursor: pointer;
}
```

that'd be great! I missed the `cursor: pointer` part 😆

Reply:
looks really cool with the red highlight on hover + pointer 💯